plugin/eqdetect/source/dextool/plugin/eqdetect/subfolder/codegenerator.d

```diff
@@ -51,4 +51,48 @@ class SnippetFinder{
         temp = temp ~ content[mutation.offset_end .. content.length];
         return temp;
     }
+
+    @trusted static auto generateKlee(string[] params, string source_name, string mutant_name, string function_name){
```

Comment:
tip: create types for the parameters to add that the programming language can understand beyond "string" and the name.
mutant_name is strictly different from function_name.

source_name could be an AbsolutePath.
mutant_name could be a:
```d
struct MutantName {
   string value;
   alias value this;
}
```
function_name could be a CFunctionName from source/cpptooling/type.d.

This gives you clarity, ability to see in the type system *how* the data is used, easier to refactor etc.

tip: Kevlin Henneys video about stringly typed systems. [ Kevlin Henney - Seven Ineffective Coding Habits of Many Programmers 26min+](https://www.youtube.com/watch?v=ZsHMHukIlJY)